src/Endo/README.md

```diff
@@ -0,0 +1,247 @@
+# Endo
+
+```haskell
+Endo a
+```
+
+`Endo` is a `Monoid` that will combine (2) functions that have matching domains
+and codomains (endofunctions) under function composition. Due to the nature
+of `Endo` wrapping a function, the underlying value can either be extracted
+using [`valueOf`](#valueof) like any other `Monoid` or can be executed using
```

Comment:
`can be executed using directly using` should be `can be executed directly using`